content/en/blogs/transform-traditional-applications-into-microservices.md

```diff
@@ -0,0 +1,227 @@
+---
+title: 'Refactor Traditional Applications into Microservices to Enable the Traffic Monitoring Feature'  
+tag: 'KubeSphere, Kubernetes, Microservices'  
+keywords: 'KubeSphere, Kubernetes, Microservices, Traffic Monitoring, Service Mesh'  
+description: 'This article describes how to refactor a traditional application into microservices to use service mesh features, such as grayscale release, traffic monitoring, and tracing.'   
+createTime: '2021-07-07'  
+author: 'Zackzhang, Bettygogo'  
+snapshot: '/images/blogs/transform-traditional-applications-into-microservices/traffic-monitoring-cover.png'
+---
+
+## Challenges
+
+When users are using the service mesh feature of KubeSphere, most of them merely know how to deploy the Bookinfo sample, but have no idea about how to use microservices, let alone refactor traditional applications into microservices.
```

Comment:
When trying to use the service mesh features, most of KubeSphere users only manage to deploy a Bookinfo sample on KubeSphere. They are also struggling in understanding how to explore the full capabilities of service mesh, let alone refactor traditional applications into microservices.